spec/mailers/invitation_mailer_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe InvitationMailer do
+  describe '#invitation_email' do
+    subject { described_class.invitation_email(params) }
+
+    context 'valid params' do
+      let(:params) do
+        {
+          requester_email: 'some@doctor.dev',
+          respondent_email: 'some@patient.dev',
+          response_uuid: 'some_uuid'
```

Comment:
Misschien nog even vervangen door `SecureRandom.uuid` om te kijken of het genereren van een link met een echte uuid er in wel slaagt.